ext/standard/tests/checkdnsrr.phpt

```diff
@@ -1,14 +1,12 @@
 --TEST--
 bool checkdnsrr ( string $host [, string $type = "MX" ] );
---CREDITS--
-marcosptf - <marcosptf@yahoo.com.br> - @phpsp - sao paulo - br 
```

Comment:
Hello, if `--CREDITS--` aren't deprecated yet, I think it might be much more contributors friendly to leave them in...

Reply:
Sure, reverted that change